Mergin/validation.py

```diff
@@ -310,6 +320,12 @@ def check_snapping(self):
                 # snapping in Input using QGIS setting is disabled but project has snapping activated
                 self.issues.append(MultipleLayersWarning(Warning.MERGIN_SNAPPING_NOT_ENABLED))
 
+    def check_datum_shift_grids(self):
+        grids = get_datum_shift_grids()
+        for grid in grids.keys():
+            if not os.path.exists(os.path.join(self.qgis_proj_dir, "proj", grid)):
+                self.issues.append(MultipleLayersWarning(Warning.MISSING_DATUM_SHIFT_GRID))
```

Comment:
do we need multiple warnings if there are more grids missing?

Reply:
I don't think we need many identical warnings in this case. We do the same in some other cases, show only one waring for similar issues.